proto/nexus/exported/v1beta1/types.proto

```diff
@@ -15,6 +15,7 @@ message Chain {
   string native_asset = 2;
   bool supports_foreign_assets = 3;
   tss.exported.v1beta1.KeyType key_type = 4;
+  string module = 5;
```

Comment:
I don't really like putting the module here, but so far I have not found a better solution. using interfaces is clunky because you can't embed structs. Any other ideas how to do this?

Reply:
one solution is to check link msgs in ante handler, and route there. still not elegant